src/components/input/input-select.jsx

```diff
@@ -1,10 +1,11 @@
 import React, { PropTypes } from 'react';
+import classNames from 'classnames';
 import { kebabCase } from 'lodash';
 import InputDefault from './input-default';
-// import variables from '../../utilities/variables';
-import Icon from '../icon/icon';
-// TODO: Move SCSS into JSS
-// import './input-select.scss';
+import IconChevronUp from '../icon/icons/chevronUp';
+import IconChevronDown from '../icon/icons/chevronDown';
+import styleSheet from './input-select-styles';
+import color from '../../styles/color';
```

Comment:
Would be better to receive color through theme. You have access to the `theme` inside styleManager (`styleManager.theme`)